src/components/CallView/shared/ViewerOverlayCallView.vue

```diff
@@ -277,7 +293,7 @@ export default {
 }
 
 .viewer-overlay__video {
-	position: relative;
-	height: 100%;
+  position: relative;
+  height: 100%;
```

Comment:
https://github.com/nextcloud/server/blob/master/.editorconfig#L8-L9

That is not our intended coding style btw.
I guess the config is not picked up by your editor because you work directly in the talk repo instead of having a project with nextcloud and the app inside?
Should we copy it to talk?